ext/fetch/lib.rs

```diff
@@ -197,22 +185,27 @@ pub struct FetchReturn {
 #[op]
 pub fn op_fetch<FP>(
   state: &mut OpState,
-  args: FetchArgs,
+  method: ByteString,
+  url: String,
+  headers: Vec<(ByteString, ByteString)>,
+  client_rid: Option<u32>,
+  has_body: bool,
+  body_length: Option<u64>,
   data: Option<ZeroCopyBuf>,
```

Comment:
That's a lot of arguments :P I'm not convinced this is preferable to having an "option bag", but if it results in fewer LLVM lines (and potentially faster compile) then we should go with it.